hbase-client/src/main/java/org/apache/hadoop/hbase/client/CheckAndMutateResult.java

```diff
@@ -27,6 +27,8 @@ public class CheckAndMutateResult {
   private final boolean success;
   private final Result result;
 
+  private QueryMetrics metrics = null;
```

Comment:
Sadly, because the `Result` can be null, we need to stash the `QueryMetrics` separately.